ci/autoresearch/gpio.py

```
@@ -59,11 +59,50 @@ async def run_gpio_pretest(
+                        port,
+                        timeout=timeout,
+                        serial_interface=serial_interface,
+                        verbose=True,
+                    )
+                    await client.connect(boot_wait=3.0, drain_boot=True)
+                    ping_response = await client.send("ping", retries=3)
+                    print(f"\u2705 Ping successful after DTR reset: {ping_response.data}")
+                except KeyboardInterrupt as ki:
+                    handle_keyboard_interrupt(ki)
+                    raise
+                except Exception as e2:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)